### PR TITLE
feat: manager adjustment dashboard

### PR DESCRIPTION
**Pull request by mdeljavan**

### CODE REVIEW

File: src/containers/main/MainRoutes.tsx [hunk 1]
```diff
@@ -127,7 +134,7 @@ export function MainRoutes(props: FCProps<Props>) {
       <Switch>
         <Redirect exact path="/" to="/peer-review" />
         <Route path="/peer-review/:uid/:tab?" children={<PeerReviewPage />} />
-        <Route path="/peer-review/" children={<PeerReviewBoardPage />} />
+        <Route path="//peer-review/" children={<PeerReviewBoardPage />} />
```
Comment (smmoosavi) [inline src/containers/main/MainRoutes.tsx:137]: این درسته؟

File: src/pages/manager-adjustment-dashboard/useFilters.ts [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+  };
+
+  const filterManager = (row: Row) => {
+    if (filters.manager) {
+      return row.manager.includes(filters.manager);
+    }
+    return true;
+  };
+
+  const filterStatus = (row: Row) => {
```
Comment (mehrdad-e) [inline src/pages/manager-adjustment-dashboard/useFilters.ts:35]: IMO use switch

File: src/pages/manager-adjustment-dashboard/TableFilters.tsx [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+          value={filters.user}
+          variant="outlined"
+        />
+      </Grid>
+      <Grid item md={4}>
+        <TextField
+          fullWidth
+          label={i18n._('Manager')}
+          margin="dense"
+          onChange={(e) => {
```
Comment (mehrdad-e) [inline src/pages/manager-adjustment-dashboard/TableFilters.tsx:39]: IMO don't write inline

File: src/pages/manager-adjustment-dashboard/ManagerAdjustmentDashboardTable.tsx [hunk 4]
```diff
@@ -0,0 +1,171 @@
...
+    return 'complete';
+  }
+  if (value <= 20) {
+    return 'low';
+  }
+  if (value < 60) {
+    return 'medium';
+  }
+  return 'high';
+};
```
Comment (mehrdad-e) [inline src/pages/manager-adjustment-dashboard/ManagerAdjustmentDashboardTable.tsx:150]: Maybe switch?

**Review by mehrdad-e — APPROVED** [review]

**Pull request merged**